Policies/github-acceptable-use-policies.md

```diff
@@ -9,66 +9,69 @@ topics:
+- [harasses or abuses](/github/site-policy/github-bullying-and-harassment) another individual or group, including our employees, officers, and agents, or other users;
 
-- directly supports [unlawful active attack or malware campaigns](/github/site-policy/github-community-guidelines#active-malware-or-exploits) that are causing technical harms — such as using our platform to deliver malicious executables or as attack infrastructure, for example by organizing denial of service attacks or managing command and control servers — with no implicit or explicit dual-use purpose prior to the abuse occurring; or
+- [threatens or incites violence](/github/site-policy/github-threats-of-violence-and-gratuitously-violent-content) toward any individual or group, especially on the basis of who they are; 
 
-- shares unauthorized product licensing keys, software for generating unauthorized product licensing keys, or software for bypassing checks for product licensing keys, including extension of a free license beyond its trial period; or
+- [gratuitously depicts or glorifies violence](/github/site-policy/github-threats-of-violence-and-gratuitously-violent-content), including violent images; or
 
-- infringes any proprietary right of any party, including patent, trademark, trade secret, copyright, right of publicity, or other right.
+- is off-topic, or interacts with platform features in a way that significantly or repeatedly [disrupts the experience of other users](/github/site-policy/github-disrupting-the-experience-of-other-users).
 
-Please see our [Community Guidelines](/github/site-policy/github-community-guidelines#what-is-not-allowed) for more details.
 
-## 3. Conduct Restrictions
-While using the Service, under no circumstances will you:
+## 3. Intellectual Property, Authenticity, and Private Information
+We do not allow content or activity on GitHub that:
 
-- [harass, abuse](/github/site-policy/github-community-guidelines#bullying-and-harassment), [threaten, or incite violence](/github/site-policy/github-community-guidelines#threats-of-violence) towards any individual or group, including our employees, officers, and agents, or other users;
+- infringes any proprietary right of any party, including patent, trademark, trade secret, copyright, right of publicity, or other right;
 
-- post off-topic content, or interact with platform features, in a way that significantly or repeatedly [disrupts the experience of other users](/github/site-policy/github-community-guidelines#disrupting-the-experience-of-other-users);
+- shares unauthorized product licensing keys, software for generating unauthorized product licensing keys, or software for bypassing checks for product licensing keys, including extension of a free license beyond its trial period;
```

Comment:
I believe there should be an exemption here for interoperability reasons. For example the source of hook dlls to remove invasive DRM which causes games to no longer run on current versions of windows should be allowed.

Reply:
We pushed https://github.com/github/site-policy/pull/528/commits/ddd9c0f14d552292cd5f5b0a94bf3c962bc3fbbd to add `unlawfully` before `shares` to clarify that we don’t intend for this restriction to apply where a legal exemption, such as for the purpose of interoperability, may apply.